catalyst/rl/offpolicy/trainer.py

```diff
@@ -340,13 +222,14 @@ def train(self):
             start_time = time.time()
 
     def update_target_weights(self, step_index):
-        if step_index % self.actor_update_period == 0:
-            self.algorithm.target_actor_update()
-            self.actor_updates += 1
+        if not self.discrete_actions:
```

Comment:
`not self.discrete_actions` sounds a bit confusing, it's more about `actor-critic` logic then.

Reply:
For now it seems to me that we need a flag which indicates whether the action space is continuous or discrete. Probably, we can include it inside the environment wrapper as you suggested before.